tools/deployment/make_windows_package.ps1

```diff
@@ -605,17 +606,30 @@ function Main() {
     Get-Help
   }
 
-  $scriptPath = Get-Location
-  $buildPath = Join-Path $scriptPath 'build\windows10\osquery\Release'
-  $daemon = Join-Path $buildPath 'osqueryd.exe'
-  $shell = Join-Path $buildPath 'osqueryi.exe'
+  $buildPath = ''
+  if ($Build -eq 'buck') {
+    $buildPath = Join-Path $osqRoot 'buck-out\release\gen\osquery'
+  } else {
+    $buildPath = Join-Path $osqRoot 'build\windows10\osquery\Release'
```

Comment:
This makes an assumption about where `cmake ..` was run. Is there a more flexible way to discover this path?

Reply:
Perhaps instead of taking in `BuildType` it can take `buildPath` directly?

Reply:
Sure, I'll get this fixed up tonight, sorry for the delay!